cvat-sdk/cvat_sdk/core/client.py

```diff
@@ -354,13 +404,51 @@ def make_endpoint_url(
 
 
 def make_client(
-    host: str, *, port: Optional[int] = None, credentials: Optional[tuple[str, str]] = None
+    host: str,
+    *,
+    port: Optional[int] = None,
+    credentials: Union[Credentials, tuple[str, str], None] = None,
+    access_token: Optional[str] = None,
```

Comment:
How about interpreting `credentials="..."` as an access token? Then there's no need to add another parameter, and the signatures of `make_client` and `Client.login` can be consistent with each other.